package.json

```diff
@@ -26,7 +26,8 @@
   "homepage": "https://github.com/mdn/browser-compat-data#readme",
   "devDependencies": {
     "ajv": "^5.0.1",
-    "mdn-confluence": "0.0.7"
+    "mdn-confluence": "0.0.7",
+    "yargs": "^11.1.0"
```

Comment:
Sorry for not having a look at this earlier, but meanwhile yargs comes with version 12.

Reply:
Well, yargs was at version 11 when I made this, and it’s version `11.1` that’s in the `node_modules` tree:
https://github.com/mdn/browser-compat-data/blob/f4f8ce8eb117281c5037d15cac257293fd19301c/package-lock.json#L2103-L2104

Reply:
Alright, let go with this version for now then.